packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableActionsDemo.tsx

```diff
@@ -72,16 +72,16 @@ export class TableActionsDemo extends React.Component<TableProps, TableState> {
     return [
       {
         title: 'Some action',
-        // tslint:disable-next-line:no-shadowed-variable
-        onClick: (event, rowId, rowData, extra) =>
-          // tslint:disable-next-line:no-console
+        // eslint-disable-next-line no-shadow
+        onClick: (event, rowId, rowData) =>
+          // eslint-disable-next-line no-console
           console.log(`clicked on Some action, on row ${rowId} of type ${rowData.type}`)
       },
       {
         title: 'Another action',
-        // tslint:disable-next-line:no-shadowed-variable
-        onClick: (event, rowId, rowData, extra) =>
-          // tslint:disable-next-line:no-console
+        // eslint-disable-next-line no-shadow
+        onClick: (event, rowId, rowData) =>
```

Comment:
Should we restore the `extra` prop in the functions above? Perhaps just rename it as _extra to avoid the warning?

Reply:
Added